### ISSUE TITLE
surface=compacted_gravel

### ISSUE THREAD

**Issue by matkoniecz** [BikeCitizens/bikecitizens-taginfo#2]
This listing shows `surface=compacted_gravel` as being supported.

This surface value was retagged in https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account/fixing_malformed_surface_tags to `surface=compacted` after discussion, as `surface=compacted` is exactly for compacted gravel.

Hopefully you agree with such change,

Sorry for not writing earlier, I have not considered that you would want notification if new value is already also supported.

Would you consider such notifications as useful?

Would you prefer to get such issue when I start considering replacement of one tag by another as a clear alias?

Or only once bot edit is officially proposed?

Or only once bot edit replacing such tag with another is running?

Or only once tag is fully gone?

Would it be different if some tag is replaced only manually without proposing bot edit?

Would you prefer separate issue for each tag? Update existing open one with additional listings? Update existing one even if it is closed?
